test/expiring-todo-comments.js

```diff
@@ -110,6 +110,22 @@ ruleTester.run('expiring-todo-comments', rule, {
 		{
 			code: '// TODO [but [it will]] [fallback] [[[ to the default ]]] rule [[',
 			errors: []
+		},
+		{
+			code: '// TODO ISSUE-123 fix later',
+			options: [{allowWarningComments: false, ignore: ['ISSUE-\\d+']}]
+		},
+		{
+			code: '// TODO [ISSUE-123] fix later',
+			options: [{allowWarningComments: false, ignore: ['ISSUE-\\d+']}]
+		},
+		{
+			code: '// TODO [2020-01-01, ISSUE-123] fix later',
+			options: [{allowWarningComments: false, ignore: ['ISSUE-\\d+']}]
+		},
+		{
+			code: '// TODO [2020-01-01, Issue-123] fix later',
```

Comment:
Can  you  use  a more  earlier date ? This  is  not  that  obvious.

Reply:
Will update.